test/Cargo.toml

```
@@ -8,6 +8,7 @@ publish = false
 [dependencies]
 anyhow = "1.0.41"
 forc = { path = "../forc", features = ["test"], default-features = false }
+forc-util = { version = "0.14.1", path = "../forc-util" }
```

Since `test` isn't published it doesn't need a version for internal deps.
```suggestion
forc-util = { path = "../forc-util" }
```